### PR TITLE
[Feature] generate input fields, syntax highlighting, light mode

### LINKED ISSUE #598: [Proposal] Auto-generate input fields from program source code on aleo.tools
## 💥 Proposal - Auto-generate input fields from program source code on aleo.tools

The `aleo-wasm` toolkit provides the ability to get a javascript object of the inputs for a program.

When a user enters a program on aleo.tools - a webform should be created that has fields for all the inputs to that program. Users should then be able to enter their inputs to the program and submit it to execute the program. The form should allow the user to choose which function they want to execute

The form should also:
* Validate the inputs are valid and of the correct type prior to submission
* Display a useful error message if the inputs are incorrect
* Allow users to clearly select the function within the program they want to execute

### LINKED ISSUE #610: [Proposal] Implement Syntax Highlighting in aleo.tools
## 💥 Proposal - Implement Syntax Highlighting in aleo.tools

When a user views a program on aleo.tools, the text box should be able to recognize Aleo instruction syntax and apply appropriate coloring and styling to the code elements.

### PR DESCRIPTION
**Pull request by onetrickwolf**
## Motivation
Resolves #598
Resolves #610 

## Test Plan

Preview URL: https://auto-gen-preview--aleo-tools-3481ab.netlify.app/develop

- Go to `/develop` and use new "Execute Program" card. Functionality from the original execute should be there in addition to inputs auto generating and general style / validation improvements and changes

- Typing in a program without `.aleo` and hitting enter or the search icon should automatically fill out the `.aleo` portion of the program ID

- Can generate private key directly in form

- Demo dropdown can be selected to populate hello demo

- Note syntax highlighting should work for program input

- Dark mode or light mode which can be toggled in the side bar

## Video

https://github.com/AleoHQ/sdk/assets/13836477/9de28f38-0082-4b9b-9551-13d52dfbeaa7

### CODE REVIEW

File: website/src/tabs/develop/execute/CodeEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import { useState } from "react";
+
+const aleoSyntaxHighlight = {
+    start: [
+        {
+            regex: /(?:^|\s)(function|program|as|by|interface|closure|into|import)(?:$|\s)/,
+            token: "keyword",
+        },
+        {
+            regex: /(?:^|\s)(finalize|mapping|increment|decrement)(?:$|\s)/,
```
Comment (collinc97) [inline website/src/tabs/develop/execute/CodeEditor.jsx]: These opcodes are deprecated in aleo instructions.
I have added an equivalent issue on `aleo-docs` to remove them from the syntax highlighter.
```suggestion
            regex: /(?:^|\s)(finalize|mapping)(?:$|\s)/,
```

File: website/src/tabs/develop/execute/index.jsx [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const layout = { labelCol: { span: 4 }, wrapperCol: { span: 18 } };
+
+export const Execute = () => {
+    const [form] = Form.useForm();
+    const aleoWASM = useAleoWASM();
+
+    const demoSelect = async (value) => {
```
Comment (collinc97) [inline website/src/tabs/develop/execute/index.jsx:28]: This is much better than the current demo button!

**Review by iamalwaysuncomfortable — APPROVED** [review]
I'd say let's make the manual inputs off by default, but other than that, this is great, thank you so much!

**Review by collinc97 — APPROVED** [review]
Thanks for addressing feedback - looks great to me!

**Pull request merged**